ui/src/overview.tsx

```diff
@@ -134,6 +140,23 @@ function ParticipantCriteria(props: { group: Group; criteria: Criteria }) {
   const cohort = useCohort();
   const dispatch = useAppDispatch();
 
+  const [dialog, showRenameCriteria] = useDialog({
```

Comment:
Rename these to `renameDialog, showRenameDialog` so it's more obvious they're related.